tests/net/lib/lwm2m/functional/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.20.0)
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(lwm2m_client)
```

Comment:
Should we use a more descriptive name for the project, like `lwm2m_interop_tests`?